### ISSUE TITLE
iOS - image orientation not written to EXIF

### ISSUE THREAD

**Issue by ramapongithub** [mapillary/mapillary_issues#2386]
### Basic information
* Platform: iPhone 6 iOS 10.0.1
* Version: Mapillary 4.5.6

When looking further into [Issue 2383](https://github.com/mapillary/mapillary_issues/issues/2383) I tried to make a sequence and write the data to EXIF. Purpose was to see if the same image orientation shifting happens there. But no image orientation was written to the EXIF data. Tested with Geosetter/exiftool and Windows file information.
Instead there was raw data in the subject/title fields. In this were amongst other things "TrueHeading" and "MagneticHeading", but not consistent with the true image orientation.

**Comment by millenbop** [comment]
Hi! Did you press the top-left button in the Uploads view so that the app adds EXIF tags before copying the photos over to your computer?

**Comment by ramapongithub** [comment]
Yes, that is what I did. Else there wouldn't even be the positions in the EXIFs.

I learned from a near fatal mistake some time ago. The behavior of the program was changed and I didn't realised it. So I went on an walk, copied the images via iTunes (horrible program!) to the pc and deleted them on the phone. Then I started the upload script as usally and .... stared on the screen in shock. No EXIF data!
Luckilly I had the GPSr running in parallel and could use the gpx-track to restore position and orientation from the timestamps.
